test/route.js

```diff
@@ -480,7 +500,7 @@ describe('Router', function () {
           .expect(200, cb)
         })
 
-        it('should work in a named parameter', function (done) {
+  /*      it('should work in a named parameter', function (done) {
```

Comment:
Yeah, it does depend on path-to-regexp 1.x so it couldn't be landed until that dependency is updated and I guess this test would be deleted or rewritten as part of that.